dashboard/test/controllers/api/v1/pd/applications_controller_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+module Api::V1::Pd
+  class ApplicationsControllerTest < ::ActionController::TestCase
+    setup_all do
+      regional_partner = create :regional_partner
+      application = create :pd_facilitator1819_application, regional_partner_id: regional_partner.id
```

Comment:
Is the regional partner used? I think this is extraneous.

Also note this won't work once I merge https://github.com/code-dot-org/code-dot-org/pull/18519 because the partner will be matched on save based on the form_data program. See https://github.com/code-dot-org/code-dot-org/pull/18519/files#diff-d5b8fcfde3bd10374964117e30758981R45

Reply:
It was used to get the regional partner name in the serializer, but since I'm refactoring that it won't be needed in the future.